<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Combatscreen</class>
 <widget class="QWidget" name="Combatscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Combatscreen</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Ability01">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ability02">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ability03">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="RangerWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="ClericWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="WizardWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="WarriorWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="DragonWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>401</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="clericHealthBar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="rangerHealthBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="wizardHealthBar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="warriorHealthBar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="dragonHealthBar">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>500</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
